drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPage.java

```diff
@@ -76,13 +78,29 @@ public void prepareView() {
 
         setupWarningMessages();
         setupOperator();
+        setupCurrentField();
     }
 
-    private void setupOperator() {
+    void setupCurrentField() {
+        view.setCurrentField(currentField());
+        view.currentFieldToggle(hasFactField());
+    }
+
+    String currentField() {
```

Comment:
A great example of an alternative way to achieve "a : b" construction (with null/empty checks) I've seen :-)